packages/kotti-table/src/TableColumn.js

```diff
@@ -63,16 +63,21 @@ const TableColumn = {
 		cellClass: updateColumnsfor('cellClass'),
 		prop: updateColumnsfor('prop'),
 		hidden: updateColumnsfor('hidden'),
+		order: updateColumnsfor('order'),
 		default: updateColumnsfor('default'),
 	},
 	mounted() {
-		this.columnConfig.index =
-			this.columnConfig.index || [].indexOf.call(this[KT_TABLE].$children, this)
-		this[KT_STORE].commit('insertColumn', this.columnConfig)
+		const columnIndex = [].indexOf.call(this[KT_TABLE].$children, this)
```

Comment:
Why is this `.call` necessary?

Reply:
It would if $children isn't a proper array but a DOMNode array

But I don't think $children is I'll have to test it, I had taken that part originally as is form element ui when I was looking at other libraries for their solutions

will review